README.md

```diff
@@ -41,13 +41,14 @@ This library uses semantic versioning and follows Okta's [Library Version Policy
 | Version | Status                           |
 | ------- | -------------------------------- |
 | 0.x     | :warning: Beta Release (Retired) |
-| 1.x     | :heavy_check_mark: Release       |
+| 1.x     | :warning: Retired                |
+| 2.x     | :heavy_check_mark: Release       |
```

Comment:
Is this PR causing a major version release, and if so, why?

Reply:
This PR doesn't cause any major version release. README has been updated for two reasons:
1) additional commit to trigger CI checks
2) README was outdated, as we have already released SDK 2.x https://pypi.org/project/okta/, looks like this a good PR to fit into, but I can make a separate PR if needed